frontend/src/metabase/visualizer/components/DataImporter/DatasetsList/DatasetsListItem.tsx

```diff
@@ -1,10 +1,10 @@
-import cx from "classnames";
 import { useMemo } from "react";
 
 import { useGetCardQuery } from "metabase/api";
+import ButtonGroup from "metabase/core/components/ButtonGroup";
```

Comment:
IIRC `metabase/core/components` are mostly deprecated, could we use something else? I'm not sure if it's doing a lot of styling actually since internally it adds styles to `metabase/core/components/Button`, and we use the `metabase/ui` button component

Reply:
I don't think this comonent exists in another place, did you have something in mind? Or should I add it myself?